CDBTo3DTiles/src/CDBModels.cpp

```diff
@@ -630,6 +636,12 @@ CDBGSModels::FindGSModelTexture::FindGSModelTexture(const std::string &GSModelTe
     , m_GSModelTextureTileName{GSModelTextureTileName}
 {}
 
+CDBGSModels::FindGSModelTexture::~FindGSModelTexture() noexcept
+{
+    // Eh OSG doesn't close the archive after ref_ptr is released, so we do it ourselves
```

Comment:
```suggestion
    // OSG doesn't close the archive after ref_ptr is released, so we do it ourselves
```

Is "Eh" supposed to be in the comment?